persistent/cPersistence.c

```diff
@@ -1270,6 +1271,18 @@ Per_get_mtime(cPersistentObject *self)
         return Py_None;
     }
 
+    if (!TimeStamp)
+    {
+        PyObject* ts_module;
+        ts_module = PyImport_ImportModule("persistent._timestamp");
```

Comment:
This had to move from module initialization time because it created a cycle.

Note that it's importing the C version only now. 

Also note the new error handling at the end (`if !TimeStamp`). That turned out to be a problem that resulted in a `SystemError` at one point; that shouldn't be possible now that we're getting the C implementation, however.